tests/MageTest/Task/FactoryTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace MageTest\Task;
+
+use Mage\Task\Factory;
+use PHPUnit_Framework_TestCase;
+
+/**
+ * @group Mage_Task
+ * @coversDefaultClass Mage\Task\Factory
+ */
+class FactoryTest extends PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        require_once __DIR__ . '/MyTaskExample.php';
+    }
+
+    /**
+     * @covers Mage\Task\Factory
+     */
+    public function testLoadFromExternalNamespace()
+    {
+        $factory = new Factory;
+
+        $configMock = $this
+            ->getMockBuilder('Mage\Config')
+            ->getMock();
+
+        $myTask = $factory->get('My\Task\Example', $configMock);
```

Comment:
I'm interested in one thing. Since tasks can be defined with namespace, so please test the following:
* Remove `MyTaskExample` class
* Create a mock of this task, but without namespace (PHPUnit's bug doesn't allow to name classes with namespaces):
```php
$this->getMockBuilder('Mage\Command\AbstractCommand')
    ->setMockClassName('MyTaskExample')
    ->getMock();
```
* Try to get your class with factory by:
```php
$myTask = $factory->get('MyTaskExample');
```
* and finally assert:
```php
$this->assertInstanceof('MyTaslExample', $myTask);
```

Please try this thing and tell me how it went :) Thanks!